paymentsheet/src/main/java/com/stripe/android/paymentsheet/LinkHandler.kt

```diff
@@ -22,7 +23,8 @@ import kotlinx.coroutines.flow.flatMapLatest
 import javax.inject.Inject
 
 internal class LinkHandler @Inject constructor(
-    val linkLauncher: LinkPaymentLauncher,
+    private val linkLauncher: LinkPaymentLauncher,
+    private val linkConfigurationCoordinator: LinkConfigurationCoordinator,
```

Comment:
Hopefully we can model this better so that we don't need to inject it here and in the view models.